api/init/api.py

```diff
@@ -1,10 +1,10 @@
-from batch import Batch
-from data_source import DataSource
-from flask import abort, Blueprint, Flask, jsonify, request, url_for
-from flask_restplus import Api, Resource, fields
 import logging
+import os
 import sys
-import utils
+from flask import Blueprint, Flask, url_for
+from flask_restplus import Api
+from health.routes import register_health
+from graphqlapi.routes import register_graphql
```

Comment:
@ThomasGassmann 
Can we rename this folder/module into `graphql` rather than `graphqlapi`?
Just for simplicity...

Reply:
The new package I've added already includes a module called `graphql`. Any other naming suggestions? @alexisrolland